spec/_superseded_operations.yaml

```diff
@@ -532,14 +562,10 @@ $schema: ../json_schemas/_superseded_operations.yaml
     - GET
     - POST
 /_opendistro/_security/kibanainfo:
-  superseded_by: /_plugins/_security/kibanainfo
+  superseded_by: /_plugins/_security/dashboardsinfo
   operations:
     - GET
     - POST
-/_opendistro/_security/sslinfo:
-  superseded_by: /_plugins/_security/sslinfo
-  operations:
-    - GET
```

Comment:
Is this entry being removed because it's now out of sync with `/_plugins/_security/sslinfo`?

Reply:
we didn't support `_plugins/_security/sslinfo`. only `_opendistro/..` is supported currently. See [here](https://github.com/opensearch-project/security/blob/905c97d4fed185362a12c4b46aa56ad8febabb4d/src/main/java/org/opensearch/security/ssl/rest/SecuritySSLInfoAction.java#L50).